easybuild/easyconfigs/d/Dycore/Dycore-551f6b4-CrayGNU-18.08-cordex-double.eb

```diff
@@ -0,0 +1,39 @@
+# omlins, kraushm, jg, havogt (cscs)
+easyblock = 'CMakeMake'
+
+name = 'Dycore'
+version = '551f6b4'
+implementation = 'cordex'
+precision = 'double'
+versionsuffix = '-%s-%s' % (implementation, precision)
+
+homepage = 'https://github.com/C2SM-RCM/cosmo-pompa/tree/master/dycore (-b crclim)'
+description = """cosmo-pompa-dycore"""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'verbose': False}
+# repo is private (+ no official release)
+sources = [
+    {'filename': '/apps/common/UES/easybuild/sources/d/%(name)s/'              \
+                 '%(name)s_%(version)s.tar.gz'}
```

Comment:
Can you use the same pattern also in the other recipes?

Reply:
Done.